src/RCTAsyncStorage.ts

```diff
@@ -2,15 +2,18 @@
 import { NativeModules, TurboModuleRegistry } from 'react-native';
 import { shouldFallbackToLegacyNativeModule } from './shouldFallbackToLegacyNativeModule';
 
-let RCTAsyncStorage =
-  NativeModules['PlatformLocalStorage'] || // Support for external modules, like react-native-windows
-  NativeModules['RNC_AsyncSQLiteDBStorage'] ||
-  NativeModules['RNCAsyncStorage'];
+// TurboModuleRegistry falls back to NativeModules so we don't have to try go
+// assign NativeModules' counterparts if TurboModuleRegistry would resolve
+// with undefined.
+let RCTAsyncStorage = TurboModuleRegistry
```

Comment:
I wonder if we should just bump the minimum requirement to 0.61 so we don't have to do this check. What do you think @krizzu?

Reply:
I'm fine with the change, although that means we'd have to keep a compatibility table in readme (so that, we show which version is supported by which RN version). I guess it's safe to say we support 0.60 and up version

Reply:
Okay, let's do this after this PR merges. Can you fix the Release job? It looks like it's unable to fetch user and password for some reason.

Reply:
@tido64 sorry, missed the message
I created PR to separate release from Pull Request checks:
https://github.com/react-native-async-storage/async-storage/pull/972

The PRs from external contributor fails, because of nature of "pull_request" trigger preventing access to Secrets and limiting GH token to READ only

Reply:
Ah, I didn't realize this was an issue. Thanks for looking into it. I guess that means we can merge this now?

Reply:
Yeah, let's get this in 🙏